scripts/.util/git.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+# shellcheck source=SCRIPTDIR/print.sh
+source "$(dirname "${BASH_SOURCE[0]}")/print.sh"
+
+function util::git::token::fetch() {
+  if [[ -z "${GIT_TOKEN:-""}" ]]; then
+    util::print::title "Fetching GIT_TOKEN"
+
+    GIT_TOKEN="$(
+      lpass show Shared-CF\ Buildpacks/concourse-private.yml \
+        | grep buildpacks-github-token \
+        | cut -d ' ' -f 2
+    )"
```

Comment:
This feels very team specific and not very friendly to others. Could we look into other alternatives?

Reply:
The other alternative is to provide your own `GIT_TOKEN` and not use the `--use-token` flag. This matches what we do with our `scripts/integration.sh` for language families and implementations. I guess it's a similar problem as what we have going on with https://dashboard.paketo.io/ requiring the user to supply a token.